chapters/annotations.tex

```diff
@@ -1440,27 +1486,31 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
 inner replaceable World world;
 \end{lstlisting}
 \end{example}
+\end{semantics}
+\end{annotationdefinition}
 
-A simple type or component of a simple type may have:
-\begin{lstlisting}[language=modelica]
-/*literal*/ constant Boolean absoluteValue = true;
-\end{lstlisting}%
-\annotationindex{absoluteValue}
-
+\begin{annotationdefinition}[absoluteValue]
+\begin{synopsis}\begin{lstlisting}
+/*literal*/ constant Boolean absoluteValue;
```

Comment:
```suggestion
/*literal*/ constant Boolean absoluteValue = true;
```
Should keep the default.

Reply:
In 3.6 it wasn't clear that there was a default, and I consider the specification of default an accident of the change of notation.  Hence, I think we should open a separate discussion about how to describe the default.

Reply:
This is now the topic of #3631.